app/utils/post/index.ts

```diff
@@ -152,7 +152,7 @@ export async function persistentNotificationsConfirmation(serverUrl: string, val
             });
             description = intl.formatMessage({
                 id: 'persistent_notifications.confirm.description',
-                defaultMessage: '@mentioned recipients will be notified every {interval, plural, one {minute} other {{interval} minutes}} until they’ve acknowledged or replied to the message.',
+                defaultMessage: `${ 'R' ? channelType === General.DM_CHANNEL : '@mentioned r' }ecipients will be notified every {interval, plural, one {minute} other {{interval} minutes}} until they’ve acknowledged or replied to the message.`,
```

Comment:
we cannot have if statements in translatable strings...

Reply:
Oh true, fixing now.